src/Page/GetHelp.elm

```diff
@@ -32,24 +59,70 @@ columnStyle =
         ]
 
 
-card : String -> String -> String -> Html msg
-card title linkName linkHref =
-    div [ css [ cardStyle ] ]
+
+-- There is currently only one quote in the copy, so a second paragraph is only displayed if one is added
```

Comment:
I don;t even think we need to mention, nor cater for that.

Reply:
Yeah, a relic of trying to put a list in I think.